explore/src/main/scala/explore/targeteditor/ElevationPlotNight.scala

```diff
@@ -428,15 +428,17 @@ object ElevationPlotNight {
           // Include the size in the key
           ResizingChart(options).withKey(s"$props-$resize").when(resize.height.isDefined),
           <.div(ExploreStyles.MoonPhase)(
-            <.span(
-              MoonPhase(
-                phase = moonPhase,
-                size = 20,
-                border = "1px solid black",
-                darkColor = "#303030"
-              ),
-              <.small("%1.0f%%".format(moonIllum * 100))
-            )
+            // Adapted from https://dev.to/thormeier/use-your-i-moon-gination-lets-build-a-moon-phase-visualizer-with-css-and-js-aih
+            <.div(ExploreStyles.MoonSphere)(
```

Comment:
Should we extract this into a separate component similar to the `MoonPhase` component? It doesn't really fit in the responsibility of this component now, I think

Reply:
Right, we could make it a component and move it to lucuma-ui

Reply:
I made it a component, I'll move it lucuma-ui later on